### ISSUE TITLE
Same derivation path diffrent adrreses

### ISSUE THREAD

**Issue by avialias** [iancoleman/bip39#171]
So I Input 24 word seed from Ledger Nano S.
I chose ETH.
All other default values in form.
I choose BIP39 tab and used custom dev path m/44'/60'/0' (MyEtherWallet dev path)
and I get my first public address same as in MyEtherWallet.
THats all good.

But when I Use 2nd tab BIP44, all other values same as before, I dont get same address as in MyEtherWallet.
Why is that?  derivation path is the same in both tabs BIP39, and BIP44.
And Ledger says that they generate key from m/44'/60' path.

Please ELI5 me..anyone...

PS: I can supply seed , and pictures....its test ledger seed and wallet...

**Comment by iancoleman** [comment]
Ledger does not follow BIP44 for ethereum

See https://github.com/ethereum/EIPs/issues/84#issuecomment-292324521

Use the BIP32 tab with path `m/44'/60'/0'`

Do not use the BIP44 tab.
